styles.css

```diff
@@ -19,4 +19,220 @@
     top: 0;
     background: #fff;
   }
+
+  .header-shapes {
+    position: absolute;
+    width: 462px;
+    height: 587.15px;
+    left: -44px;
+    top: 60px;
+  }
+
+  /* Toolbar */
+
+  .toolbar {
+    position: absolute;
+    height: 48px;
+    width: 375px;
+    left: 0;
+    top: 0;
+    border-radius: 0;
+  }
```

Comment:
- [ ] KIndly ensure That the content of the toolbar(logo and hamburger) is aligned with flexbox. This gives you a lot of options to align all your elements, plus, it's responsive 👍🏽 .